public/stylesheets/style.css

```diff
@@ -38,18 +37,45 @@ body {
 
 
 /*hr on the pdf preview page*/
+
 hr.preview {
     height: 2px;
     background-color: #000000;
 }
 
+#homeTopbar>nav {
+    margin-bottom: 0;
+}
+
 #searchBoxHome {
     width: 450px;
     margin: auto
 }
 
-@media screen and (max-width: 480px) {
+#discoverBox {
+    width: 550px;
+}
+
+@media screen and (max-width: 570px) {
     #searchBoxHome {
-      width: 300px;
+        width: 300px;
+    }
+    #discoverBox {
+        width: 350px;
+    }
+}
+
+@media screen and (max-width: 1200px) {
+    p#lyricInManyLanguages {
```

Comment:
sorry for being nitpicky. I'd like the name of the element to be lyricsInManyLanguages